frontend/app/src/locales/en.json

```diff
@@ -985,12 +985,19 @@
     "loopring": {
       "settings": "Settings",
       "delete_tooltip": "Delete your Loopring API key",
-      "description": "Used to query balances in Loopring. This API key will only be able to query the account that generated this key. ",
+      "description": "Used to query balances in Loopring. This API key will only be able to query the account that generated this key.",
       "hint": "Enter your Loopring API key",
       "label": "API key",
       "not_enabled": "The Loopring module is not enabled. Your balances won't be retrieved unless you enable the loopring module",
       "title": "Loopring"
     },
+    "opensea": {
+      "delete_tooltip": "Delete your OpenSea API key",
+      "description": "Used to query NFT information and balances from OpenSea.",
```

Comment:
Would be pretty cool if you can add a link here (and in the other ones) to show where they can get an api key. For opensea it would be here: https://docs.opensea.io/reference/request-an-api-key